browser_use/browser/context.py

```diff
@@ -346,6 +346,13 @@ async def _initialize_session(self):
 				await current_page.goto('about:blank')
 				logger.debug('🆕  Created new page: %s', current_page.url)
 
+			if self.config.no_viewport and not self.browser.config.headless:
+				# Sync the config window dimensions with the actual browser viewport size
+				actual_size = await current_page.evaluate("""() => ({ width: window.innerWidth, height: window.innerHeight })""")
+
+				self.config.window_width = actual_size['width']
+				self.config.window_height = actual_size['height']
```

Comment:
@pirate 
added above if condition that syncs the context config window dimensions states with the actual browser viewport size.